packages/payments/src/http-payments-repo.ts

```diff
@@ -0,0 +1,111 @@
+import type {PaymentsRepository, UnsubscribeCallback, UserSubscribedCallback} from "./payments-repo";
+import type {SubscriptionPlan, SubscriptionPlanCycle, UserSubscription} from "./subscription";
```

Comment:
Hey man, I thought ESLint would capture this, but let's keep it consistent with the other packages:
```suggestion
import type { SubscriptionPlan, SubscriptionPlanCycle, UserSubscription } from "./subscription";
```

Reply:
Found `"format": "prettier --ignore-path .eslintignore --write .",` in the main package.json that made this format changes.